recipes/comet-ms/build.sh

```diff
@@ -7,6 +7,7 @@ export CPPFLAGS="-I${PREFIX}/MSToolkit/include"
 
 sed -i.bak "s#gcc#${CC}#;s#g++#${CXX}#" MSToolkit/Makefile
 sed -i.bak "s#gcc#${CC}#;s#g++#${CXX}#" CometSearch/Makefile
+sed -i.bak 's#$(LIBS)#-lrt $(LIBS)#' Makefile
```

Comment:
```suggestion
sed -i.bak 's#-static##' Makefile
```

Grml.. I hate hardcoded Makefiles. Last try.. try to build dynamically, which you actually should do on conda almost always anyway.